tests/fixtures/serverlessTs/node_modules/ts-node.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports.register = () => null;
```

Comment:
I was thinking of adding this conditionally.

So first test without it (and confirm that `'TS_NODE_NOT_FOUND'` error was thrown), then create this file in test, and confirm it resolves.

Note that `fixures.cleanup` will remove created file from fixture, so you don't need to worry about cleanup

Reply:
done.  I did have to pass in "extraPaths" option to fixture.cleanup because it only tries to cleanup `.serverless`, but was easy enough and seems to work nicely.

Also had to pass in `shouldStubSpawn: true` so wouldn't resolve my global npm ts-node... was nice to already have that option available - nice job on making everything easy to use though :)